### ISSUE TITLE
BBC downloads fail

### ISSUE THREAD

**Issue by tonyaus** [GetiPlayerAutomator/get-iplayer-automator#465]
Since last week I have been unsuccessful in downloading a variety of programmes from BBC.  Prior to this I had no problems at all downloading programmes.

Not being a techie, I'm not sure what the error is, however when looking at the log it shows:
WARNING: Stream does not start with requested frame, ignoring data... 
INFO: Command exit code 2 (raw code = 512)

and then cycles through the various formats that I have selected, each time ending with:
WARNING: Failed to stream file /Users/tony/Movies/TV Shows/EastEnders/EastEnders - 5007. 2015-02-02 ((flashstd)).partial.mp4.flv via RTMP
INFO: skipping flashstd1 mode

I have also seen the following error:

INFO: Command exit code 3 (raw code = 768)
WARNING: Failed to stream file /Users/tony/Movies/TV Shows/Spiral Series 5/Spiral Series 5 - 7. Episode 7 ((flashhd)).partial.mp4.flv via RTMP

Help please?

PS: Im using OS X 10.10.2, Get iPlayer Automator 1.8.4 and accessing BBC via DNS from Unblock -US.

**Comment by dinkypumpkin** [comment]
Most likely geo-blocking is the reason, but impossible to say without a log.  Read Verbose Log page in wiki and try again.

**Comment by dinkypumpkin** [comment]
Duplicate of #464, #467
